No mgrctl on proxy

Bischoff opened the pull request:
## What does this PR change?

Do not install  mgrctl  on containerized proxy.

This PR makes most sense in conjunction with #1580 from Michael (meaning: without that other PR, you don't even reach this problem).

srbarrios reviewed (approved):
Thank you

szachovy reviewed (commented):
I didn't have time to check it manually, but is this part also addressed? https://github.com/uyuni-project/sumaform/blob/10b2b706e58a4c668d773132157edce84d5a4a73/salt/proxy_containerized/init.sls#L63

Bischoff commented:
> I didn't have time to check it manually, but is this part also addressed?
> 
> https://github.com/uyuni-project/sumaform/blob/10b2b706e58a4c668d773132157edce84d5a4a73/salt/proxy_containerized/init.sls#L63

Not sure when this code is executed in practice, but yeah, removed that line too. Thanks.
